configure

```diff
@@ -100,6 +102,32 @@ setup_git() {
   cd - >/dev/null 2>&1
 }
 
+setup_go() {
+  local user_override="${1}"; shift
+
+  if [ -n "${user_override}" ]; then
+    GOROOT="${user_override}"
+  else
+    local go="$(find_progs go)"
+    [ -n "${go}" ] && GOROOT="$(dirname "$(dirname "${go}")")"
+  fi
+
+  if [ -z "${GOROOT}" ]; then
+    info "Go not found; cannot run integration tests"
+  else
+    info "Using Go with GOROOT: ${GOROOT}"
+
+    mkdir -p .gopath/src/github.com/bazelbuild
+    rm -f .gopath/src/github.com/bazelbuild/sandboxfs
+    ln -s ../../../.. .gopath/src/github.com/bazelbuild/sandboxfs
```

Comment:
Not a fan of this. Deep relative paths are easy to get wrong and hard to review for correctness.

Reply:
Would you use an absolute path instead? (Yes, this has bitten me many times, but somehow I like the idea of making the tree "self-contained" in case it's moved. Not that it'd work though... I'm sure there would be many more things that would break in a move.)

Reply:
I would probably try to use a variable-rooted path. Then the problem condenses down to being sure that the variable is correct, which is often easier if the variable is constructed in a structured fashion.

Reply:
Tried doing this... but a for loop to construct the directory and symlink target seemed more complex and less readable. After all, it's a single symlink so I think I'll keep this as is for now.